pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -85,6 +85,9 @@ pyramid_oereb:
     # Whether to display the RealEstate_SubunitOfLandRegister (Grundbuchkreis) in the pdf extract or not.
     # Default to true.
     display_real_estate_subunit_of_land_register: true
+    # whether to display the "Certification" section in the pdf extract or not.
+    # Default to true
+    display_certification: false
```

Comment:
If you write `Default to true`, the default value should also be true :)

Reply:
It's `true`, if the value is not set. But if your default config defines it as false by default, this will cause confusion.
Do it either like the value above (display_real_estate_subunit_of_land_register) and set it to true or explain that if undefined, it will use true, and comment the `display_certification: false` out. That way people can uncomment it to disable the feature.